src/tables.rs

```diff
@@ -246,6 +245,6 @@ pub fn x520_mapped_to_nothing(c: char) -> bool {
 pub fn x520_mapped_to_space(c: char) -> bool {
     match c {
         '\u{09}' | '\u{0A}'..='\u{0D}' | '\u{85}' => true,
-        _ => c.is_separator(),
+        _ => c.is_whitespace(),
```

Comment:
Is this one correct. From the upstream doc string:
```rust
    /// Determines whether a character is a separator (Z). This is all spaces plus the Unicode line
    /// and paragraph separators.
```
From the example of `is_whitespace`, new line character `\n` is also included, but I'm not sure if "Unicode line" is something else and what is the paragraph separator